packages/viz/src/lib/basemap/create-google-map.ts

```diff
@@ -0,0 +1,51 @@
+import { CartoBaseMapError } from '../errors/basemap-error';
+
+const DEFAULT_OPTIONS: google.maps.MapOptions = {
+  mapTypeId: 'roadmap',
+  center: { lat: 0, lng: 0 },
+  zoom: 1
+};
+
+/**
+ * A helper function to create a GoogleMaps basemap on a 'map' DOM element, rendered using *Google Maps JS API*
+ *
+ * Examples:
+ * ```javascript
+ *    // Several options to create the map are allowed
+ *    const deckMap = carto.viz.createGoogleMap();
+ *    const deckMap = carto.viz.createGoogleMap('map', { mapTypeId: 'satellite'});
+ *    const deckMap = carto.viz.createGoogleMap('map', { mapTypeId: 'hybrid', zoom: 4 });
+ *    const deckMap = carto.viz.createGoogleMap('map', { mapTypeId: 'terrain', zoom: 4, center: { lng: 3, lat: 40 } });
+ * ```
+ * @export
+ * @param {Element | string} container
+ * @param {google.maps.MapOptions} googleMapsOptions
+ * @returns
+ */
+export function createGoogleMap(
+  container = 'map',
```

Comment:
Do you think that this container parameter would be better suited to be within the options parameter? I don't know if it would be common to define a custom container ID, but options are always needed because developers will want to customize their map center and so on.

Reply:
In my first shot I had it included in a new, bigger options param, and then I simplified it.
I have some doubts about simplicity from the user point of view. Let me give it a try, and we can finally aggree on it